lib/constants.ts

```diff
@@ -21,12 +21,12 @@ const AUTOCRAT_V0_1_IDL: AutocratProgram = require('./idl/autocrat_v0.1.json');
 export const AUTOCRAT_VERSIONS: ProgramVersion[] = [
   {
     label: 'V0.1',
-    programId: new PublicKey('metaX99LHn3A7Gr7VAcCfXhpfocvpMpqQ3eyp3PGUUq'),
+    programId: new PublicKey('FuTPR6ScKMPHtZFwacq9qrtf9VjscawNEFTb2wSYr1gY'),
     idl: AUTOCRAT_V0_1_IDL,
   },
   {
     label: 'V0',
-    programId: new PublicKey('meta3cxKzFBmWYgCVozmvCQAS3y9b3fGxrG9HkHL7Wi'),
+    programId: new PublicKey('FuTPR6ScKMPHtZFwacq9qrtf9VjscawNEFTb2wSYr1gY'),
```

Comment:
These don't change, old versions are deployed and always deployed. :)

V3 may be something different, but on devnet that is the value for `V0.3`

We now have `V0`, `V0.1` and `V0.2` deployed on mainnet. 

`V0 = meta3cxKzFBmWYgCVozmvCQAS3y9b3fGxrG9HkHL7Wi `

`V0.1 = metaX99LHn3A7Gr7VAcCfXhpfocvpMpqQ3eyp3PGUUq `

`V0.2 = metaRK9dUBnrAdZN6uUDKvxBVKW5pyCbPVmLtUZwtBp `

V0.3 should be the multiDAO where we will stop using one deploy for each DAO, however the migration of versions would be maintained the same way.

Reply:
Okay yeah that was my misunderstanding on this. Let me get these properly fixed.

Reply:
Fixed up now! Take a look when you get a chance.